tests/src/Interop/IJW/ManagedCallingNative/ManagedCallingNative.cs

```diff
@@ -14,7 +14,8 @@ class ManagedCallingNative
     {
         static int Main(string[] args)
         {
-            if(Environment.OSVersion.Platform != PlatformID.Win32NT)
+            // Disable running on Windows 7 until IJW activation work is complete. See dotnet/coreclr#21796
```

Comment:
I would prefer to avoid putting bug numbers in code. I am hoping the bug in question would point to this location instead.